.buildkite/pipeline.yml

```
@@ -107,4 +107,42 @@ steps:
     depends_on:
       - lint
       - unit
-      - test_network
\ No newline at end of file
+      - test_network
+  - label: test_byoc_aws
+    key: test_byoc_aws
+    if: build.pull_request.labels includes 'byoc-ready'
+    command: make test_byoc_aws
```

```suggestion
    command: |
      export AWS_ACCESS_KEY_ID=$$DEVEX_CI_CREDS_AWS_ACCESS_KEY_ID
      make test_byoc_aws
```
either set the `AWS_ACCESS_KEY_ID` like this before running command or change golang code to use `DEVEX_CI_CREDS_AWS_ACCESS_KEY_ID`